apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.IntegrationTests/Logging/LoggingTest.cs

```diff
@@ -256,7 +256,7 @@ public LoggerTestApplication()
 
         public virtual void ConfigureServices(IServiceCollection services)
         {
-            services.TryAddSingleton<IHttpContextAccessor, HttpContextAccessor>();
+            services.AddHttpContextAccessor();
```

Comment:
Is this a change that is specific to the .NET Core 3.0 support? It would be good to isolate the changes as much as possible - if there are parts that we can do separately, that would help.

Reply:
This method was added (by me 😄) in ASP.NET Core 2.1 and we can use this now since I've upgraded the package dependencies to 2.1 (as that is the LTS release). I could do the 2.1 upgrade along with these changes in a separate PR though.

Reply:
Right - I've just checked, and ASP.NET Core 2.0 has been out of support by MS for over a year now, so yes, let's update to 2.1 in a separate PR. Are you happy to do that, or would you like me to? (You're in a better position to know which of these changes will work in 2.1, but I realize this is my job!)

Reply:
I can do that in a separate PR. I might not have time until next week though, is that a OK?

Reply:
Fine by me - thanks in advance!